layers/+tools/eaf/packages.el

```diff
@@ -31,47 +31,16 @@
                     :repo  "manateelazycat/emacs-application-framework"
                     :files ("*")))))
 
-(defun eaf/init-ctable ()
-  (use-package ctable))
-
-(defun eaf/init-deferred ()
-  (use-package deferred))
-
-(defun eaf/init-epc ()
-  (use-package epc))
-
-;; (defun eaf/init-s ()
-;;   (use-package s))
-
 (defun eaf/init-eaf ()
   (use-package eaf
     :defer t
     :init
     (progn
-      (with-eval-after-load 'eaf
-                    (let ((eaf-apps (list 'eaf-jupyter
-                                          'eaf-browser
-                                          'eaf-airshare
-                                          'eaf-file-browser
-                                          'eaf-file-manager
-                                          'eaf-file-sender
-                                          'eaf-music-player
-                                          'eaf-system-monitor
-                                          'eaf-mindmap
-                                          'eaf-org-previewer
-                                          'eaf-terminal
-                                          'eaf-netease-cloud-music
-                                          'eaf-video-player
-                                          'eaf-js-video-player
-                                          'eaf-image-viewer
-                                          'eaf-demo
-                                          'eaf-vue-demo
-                                          'eaf-pdf-viewer
-                                          'eaf-markdown-previewer
-                                          'eaf-camera
-                                          )))
-                      (dolist (app eaf-apps)
-                        (require app nil 'noerror))))
+      (with-eval-after-load
```

Comment:
there's no point of having `with-eval-after-load 'foo` in `:init` sectiion of `(use-package foo ..)` form.

move it to `:config` and remove `with-eval-after-load`